Makefile

```diff
@@ -36,7 +36,7 @@ apply-with-kind: verify build-with-kind ## Deploy the kwok controller from the c
 		$(HELM_OPTS) \
 		--set controller.image.repository=$(IMG_REPOSITORY) \
 		--set controller.image.tag=$(IMG_TAG) \
-		--set serviceMonitor.enabled=true \
+		--set serviceMonitor.enabled=false \
```

Comment:
this will break the kind e2es since there won't be any service monitors now. Can you modify the kind-e2e to reflect this?